roles/set_custom_fact/files/device_list.fact

```diff
@@ -2960,18 +2960,24 @@
 			}
 		},
 		"kb-bsf-scaleway": {
-                        "kalite": {
-                                "activated": "True",
-                                "version": "0.16.9",
-                                "language": ["fr"]
+				"kalite": {
+					"activated": "True",
+					"version": "0.16.9",
+					"language": ["fr"]
 			},
 			"package_management": [{
 				"name": "wikiquote.fa",
 				"status": "latest"
 			}],
 			"portal": {
 				"activated": "False"
+			},
+			"vpn": [{
+				"name": "email",
+				"port": "658",
+				"master_server" : "ruminant"
```

Comment:
You don't need to define a port and a master for each VPNs.

`email`'s remote port will always be the same, its master will always be `ruminant`. And for the local port, it makes sense to use a defined port per VPN. Like, 655 for `testvpn`, 658 for `email`, and so on.

All of these could defined in a `vars/{{ item }}-vpn.yml` file included using some `with_items: {{  ansible_local.device_list[generic_project_name].vpns }}` (note the plural).